content/en/docs/contributing/blog.md

```diff
@@ -27,15 +39,28 @@ Unsuitable content includes:
 
 - Vendor product pitches
 
-To submit a blog post,
+If your blog post fits into the list of appropriate content,
 [raise an issue](https://github.com/open-telemetry/opentelemetry.io/issues/new?title=New%20Blog%20Post:%20%3Ctitle%3E)
-with the title and a short description of your blog post. If you are not a
-[Member](https://github.com/open-telemetry/community/blob/main/community-membership.md#member),
-you also need to provide a _sponsor_ for your blog post, who is a Member (by
-that definition) and who is willing to provide a first review of your blog post.
-
-If you do not raise an issue before providing your PR, we may request you to do
-so before providing a review.
+with the following details:
+
+- Title of the blog post
+- Short description and outline of your blog post
+- If applicable, list the technologies used in your blog post. Make sure that
+  all of them are open source, and prefer CNCF projects over non-CNCF projects
+  (e.g. use Jaeger for trace visualization, and Prometheus for metric
+  visualization)
+- Name of a [SIG](https://github.com/open-telemetry/community/), which is
+  related to this blog post
+- Name of a sponsor (maintainer or approver) from this SIG, who will help to
```

Comment:
If there's a neutrality concern, be mindful that  contributors might still nominate a SIG member from the same company. Of course we expect SIG members to be good citizens. :)

Reply:
Good call out, we can add an "ideally from a different company"